writecrow/Crow-Contributors
# Update README (Resolves #5)

### Purpose of Changes
This adds "Mark Fullmer" to the list of contributors. It is a nonfunctional change.

### Testing Steps
0. View the README file in this branch.
1. Verify that Mark Fullmer's name is accurate
2. Verify Mark Fullmer's title is accurate.
